app/src/main/java/org/schabi/newpipe/local/feed/FeedFragment.kt

```diff
@@ -237,33 +243,34 @@ class FeedFragment : BaseListFragment<FeedState, Unit>() {
             progressState.maxProgress == -1
 
         if (!isIndeterminate) {
-            loading_progress_text.text = "${progressState.currentProgress}/${progressState.maxProgress}"
+            feedBinding.loadingProgressText.text = "${progressState.currentProgress}/${progressState.maxProgress}"
         } else if (progressState.progressMessage > 0) {
-            loading_progress_text?.setText(progressState.progressMessage)
+            _feedBinding?.loadingProgressText?.setText(progressState.progressMessage)
```

Comment:
Why are you not using `feedBinding.loadingProgressText.` here and below?

Reply:
I was getting a `NullPointerException` when using the non-null `feedBinding` property.